src/themes/_dark.scss

```diff
@@ -0,0 +1,16 @@
+@use "../design-tokens/colors.yml";
+
+.t-dark {
+  background-color: colors.$primary-brand;
+  color: colors.$text-light;
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  a {
+    color: colors.$text-light-emphasis;
+  }
```

Comment:
Potentially premature nits:

- I wonder if links should still have a color distinguishing them or if underlines are enough?
- Are there other elements that need to change colors? `blockquote`? `li`?

(I'm fine punting on this until we need to handle it but figured I'd mention it.)

Reply:
Re: links I just looked at the screenshots again and I think they look great white. Feel free to ignore that part.

Reply:
I see this as just a starting point!